File: app/src/main/java/com/example/ftpnext/database/TableTest1/ITableTest1Schema.java
package com.example.ftpnext.database.TableTest1;

public interface ITableTest1Schema {

    String TABLE = "tabletest1";

    //Columns
    String COLUMN_DATABASE_ID = "id";
    String COLUMN_VALUE = "column_value";

    //Columns list
    String[] COLUMNS = new String[]{
            COLUMN_DATABASE_ID,
            COLUMN_VALUE,
    };

    //Create
    String TABLE_CREATE = "CREATE TABLE IF NOT EXISTS " + TABLE + " ("
            + COLUMN_DATABASE_ID + " INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "
            + COLUMN_VALUE + " INTEGER"
            + ")";
}

File: app/src/main/java/com/vpulse/ftpnext/database/TableTest1/ITableTest1Schema.kt
package com.vpulse.ftpnext.database.TableTest1

open interface ITableTest1Schema {
    companion object {
        const val TABLE: String = "tabletest1"

        //Columns
        const val COLUMN_DATABASE_ID: String = "id"
        const val COLUMN_VALUE: String = "column_value"

        //Columns list
        val COLUMNS: Array<String?> = arrayOf(
            COLUMN_DATABASE_ID,
            COLUMN_VALUE
        )

        //Create
        const val TABLE_CREATE: String = ("CREATE TABLE IF NOT EXISTS " + TABLE + " ("
                + COLUMN_DATABASE_ID + " INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "
                + COLUMN_VALUE + " INTEGER"
                + ")")
    }
}